modules/infinite_wishlist/src/Controller/WishlistController.php

```diff
@@ -87,4 +88,17 @@ public function loadProducts(Request $request)
       'products' => $products,
     ]);
   }
+
+  protected function removeLeadingBrandFromProductName($name, $brand)
+  {
+    $brandRemoved = false;
+    if (0 === strpos(strtolower($name), strtolower($brand))) {
+      $brandRemoved = true;
+      $name = trim(substr($name, strlen($brand)));
+    }
+    if ($brandRemoved && 0 === strpos($name, '-')) {
+      $name = trim(substr($name, 1));
+    }
```

Comment:
Very nice... you are trimming a '-' here, right?

What do you think about creating a loop which trims '-', ';' and ' ' (blanks)? to clean up the description and prevent cases like the one in the attached screenshot?

![bildschirmfoto 2018-02-21 um 16 22 11](https://user-images.githubusercontent.com/10079203/36488347-79d56248-1723-11e8-8de1-2279d73aec45.png)

Reply:
Boom... LGTM 👍